cmd/uber-apiserver/app/options/options.go

```diff
@@ -0,0 +1,213 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package options contains flags and options for initializing an apiserver
+package options
+
```

Comment:
the sources under uber-apiserver seem to be copied from kube-apiserver, this introduces complexity for maintenance, if we have a bugfix in this piece of code, we need to fix them in two places, should use method calls as many as as possible since those methods are already public.